File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.swift
import Foundation

func solution(_ sides:[Int]) -> Int {
    
    var sidesArr = sides.sorted()
    
    return (1...2000).filter {
        (sidesArr[1] - sidesArr[0]) < $0 && $0 < (sidesArr[1] + sidesArr[0])
    }.count
}

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.cpp
#include <algorithm>
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> sides) {
    int answer = 0;

    sort(sides.begin(), sides.end());

    for (int i = sides[1] - sides[0] + 1; i < sides[0] + sides[1]; i++) {
        answer++;
    }

    return answer;
}